### PR TITLE
[docs] Improved README #21

### LINKED ISSUE #21: [docs] Improve README
We should eliminate some redundant information from the README and link the documentation.

@lillopaco you can maybe take some inspiration from the [README of netjsonconfig](https://github.com/openwisp/netjsonconfig/blob/master/README.rst).

### PR DESCRIPTION
**Pull request by lillopaco**
Fixes #21

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -15,26 +15,27 @@ django-freeradius
...
+
+**django-freeradius** is `Django <https://www.djangoproject.com/>`_ reusable app that provides an admin interface to a `freeradius <http://freeradius.org/>`_ database.
 
-------------
 
 .. contents:: **Table of Contents**:
    :backlinks: none
    :depth: 3
 
 ------------
```
Comment (yakky) [inline README.rst]: This line must be removed

File: README.rst [hunk 2]
```diff
@@ -15,26 +15,27 @@ django-freeradius
    :target: http://badge.fury.io/py/django-freeradius
 
 ------------
+Django-freeradius is part of the `OpenWISP project <http://openwrt.org>`_.
 
-TODO:
+.. image:: http://netjsonconfig.openwisp.org/en/latest/_images/openwisp.org.svg
+  :target: http://openwisp.org
+
+**django-freeradius** is `Django <https://www.djangoproject.com/>`_ reusable app that provides an admin interface to a `freeradius <http://freeradius.org/>`_ database.
```
Comment (nemesisdesign) [inline README.rst]: "a Django reusable app"

File: README.rst [hunk 3]
```diff
@@ -15,26 +15,27 @@ django-freeradius
...
-Current features
-----------------
-
-* TODO
-
 Project goals
 -------------
 
-* TODO
+* provide a web interface to manage a freeradius database, with an additional RESTful API
```
Comment (nemesisdesign) [inline README.rst]: remove the mention of the RESTful API, we'll add that when we'll have implemented it

File: README.rst [hunk 4]
```diff
@@ -15,26 +15,27 @@ django-freeradius
...
+(using django REST framework) that will be used to retrieve session data from freeradius and to allow
+new users to register via third party apps
+* provide abstract models and admin class that can be imported and extended and reused in third party apps.
```
Comment (nemesisdesign) [inline README.rst]: should be "abstract models and admin classes"
should be "imported and extended and reused"

**Comment by nemesisdesign** [comment]
Merge manually in ed8d88c

**Pull request closed without merging**